README.md

```diff
@@ -1,4 +1,4 @@
-# Rust Compiler Performance Monitoring & Benchmarking
+# Rust Compiler Performance Monitoring & Benchmarking - An amazing project
```

Comment:
I mean: a blog post talks about how amazing this project is is helpful than self-promotion.